src/Integration/LocalServices/ErrorListInfoBarController.cs

```diff
@@ -572,10 +578,11 @@ public void QueueCheckIfUpdateIsRequired(Action<string> updateAction)
                     return;
                 }
 
-                var solutionBinding = this.host.GetService<ISolutionBindingSerializer>();
-                solutionBinding.AssertLocalServiceIsNotNull();
+                var configProvider = this.host.GetService<IConfigurationProvider>();
+                configProvider.AssertLocalServiceIsNotNull();
 
-                var binding = solutionBinding.ReadSolutionBinding();
+                // TODO: CM2: only relevant for legacy connected mode
```

Comment:
CM2?

Reply:
Short-hand for "Connected Mode version 2" - a marker to search to tag things I still need to do.